server/http.go

```diff
@@ -341,6 +392,23 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 		return appErr
 	}
 
+	storedStatusPref, appErr := p.API.KVGet(fmt.Sprintf("%v_%v", changeStatusKey, creator.Id))
+	if appErr != nil {
+		p.API.LogDebug("Could not get stored status preference from KV ", appErr)
+	}
+
+	if storedStatusPref == nil {
+		err := p.sendStatusChangeAttachment(creator.Id, p.botUserID, meetingID)
+		if err != nil {
+			p.API.LogDebug("could not send status change attachment ", "error", err)
+		}
+	}
```

Comment:
But the way we are checking is whether there is any stored status. If for some reason that is lost (DB error, or something similar) it will be also shown more times. And the Set Status function is called either way. So why not have it inside and everything is in the same place?

Reply:
That is indeed my proposal. It would mean less calls to the KVStore, less "noise" in this function, and all changing status logic being in one single place. Nevertheless, I am open to discussion if you feel it is not the best approach.

Reply:
That makes sense, it does reduce the api calls. This is a better way.